Editor/EditorCore/StripProBuilderScripts.cs

```diff
@@ -61,6 +61,7 @@ public static void Strip(ProBuilderMesh[] all)
             UnityEditor.EditorUtility.DisplayDialog("Strip ProBuilder Scripts", "Successfully stripped out all ProBuilder components.", "Okay");
 
             ProBuilderEditor.Refresh();
+            SmoothGroupEditor.Refresh();
```

Comment:
As noted above, we can avoid manually refreshing from this script by ensuring that the `MeshSelection.objectSelectionChanged` callback is invoked when scripts are removed. That way any other systems that are looking at `ProBuilderMesh` components will be notified as well.